jetty-http/src/main/java/org/eclipse/jetty/http/HttpCookie.java

```diff
@@ -87,11 +123,13 @@ public HttpCookie(String setCookie)
         _domain = cookie.getDomain();
         _path = cookie.getPath();
         _maxAge = cookie.getMaxAge();
-        _httpOnly = cookie.isHttpOnly();
+        _httpOnly = cookie.isHttpOnly() || isHttpOnlyInComment(cookie.getComment());
```

Comment:
Same again.  This contructor is using `java.net.HttpCookie`, so it has support for httpOnly and we don't need to look for it in a comment.